sound/soc/codecs/rt5682.c

```diff
@@ -34,11 +36,13 @@
 
 #define RT5682_NUM_SUPPLIES 3
 
+#if !IS_ENABLED(CONFIG_SOUNDWIRE)
```

Comment:
ah, this is a problem for Intel.
We have platforms that use rt5682 in I2S mode and we want to use it in SDW mode on others, with the same build/Kconfigs.